specs/AppNotifications/AppNotificationContentSpec/AppNotificationBuilder-spec.md

```diff
@@ -0,0 +1,1224 @@
+        AppNotificationComboBox AddItem(String id, String content);
+
+        // Adds a title to display on top
+        AppNotificationComboBox SetTitle(String value);
+
+        // Sets the default selection to be displayed by the AppNotificationComboBox
+        AppNotificationComboBox SetSelectedItem(String id);
+    };
+
+    runtimeclass AppNotificationProgressBar
+    {
+        // AppNotificationProgressBar binds to AppNotificationProgressData so the AppNotification will
+        // receive every update to the status and value. In the WinAppSDK, these binding
+        // values are static, so developers won't need to define these binding values
+        // themselves.
+        AppNotificationProgressBar();
+
+        // Setting these properties will remove the data binding with a static value
+        String Title;
+        String Status;
+        Double Value;
+        String ValueStringOverride;
+
+        AppNotificationProgressBar SetTitle(String value);
+        AppNotificationProgressBar BindTitle();
+
+        AppNotificationProgressBar SetStatus(String value);
+        AppNotificationProgressBar BindStatus();
+
+        AppNotificationProgressBar SetValue(Double value);
+        AppNotificationProgressBar BindValue();
+
+        AppNotificationProgressBar SetValueStringOverride(String value);
+        AppNotificationProgressBar BindValueStringOverride();
+    };
+
+    enum AppNotificationSoundEvent
+    {
+        Default,
+        IM,
+        Mail,
+        Reminder,
+        SMS,
+        Alarm,
+        Alarm2,
+        Alarm3,
+        Alarm4,
+        Alarm5,
+        Alarm6,
+        Alarm7,
+        Alarm8,
+        Alarm9,
+        Alarm10,
+        Call,
+        Call2,
+        Call3,
+        Call4,
+        Call5,
+        Call6,
+        Call7,
+        Call8,
+        Call9,
+        Call10,
+    };
+
+    enum AppNotificationScenario
+    {
+        Default, // The normal AppNotification behavior. The AppNotification appears for a short duration, and then automatically dismisses into Notification Center.
+        Reminder, // The notification will stay on screen until the user dismisses it or takes action.
+        Alarm,    // Alarms behave like Reminder, but alarms will additionally loop audio with a default alarm sound.
+        IncomingCall, // Incoming call notifications are displayed pre-expanded in a special call format and stay on the user's screen till dismissed.
+        Urgent, // Important notifications allow users to have more control over what 1st party and 3rd party apps can send them high-priority AppNotifications (urgent/important) that can break through Focus Assist.
+    };
+
+    enum AppNotificationDuration
+    {
+        Default, // Default value. AppNotification appears for a short while and then goes into Notification Center.
+        Long, // AppNotification stays on-screen for longer, and then goes into Notification Center.
+    };
+
+    enum ImageCrop
```

Comment:
Should this be prefix with AppNotification like everything else? (as in AppNotificationImageCrop)

Reply:
Yup. Will fix.